course_content/notebooks/matplotlib_intro.ipynb

```diff
@@ -27,49 +27,36 @@
      ],
      "language": "python",
      "metadata": {},
-     "outputs": [],
-     "prompt_number": 1
+     "outputs": []
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
       "## The matplotlib figure\n",
       "\n",
-      "At the heart of **every** matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen. Any object that can be drawn in this way is known as an \"Artist\" in matplotlib.\n",
-      "\n",
-      "Let's create our first artist using pyplot, and then show it:"
+      "At the heart of every matplotlib plot is the \"Figure\" object. The \"Figure\" object is the top level concept that can be drawn to one of the many output formats, or simply just to screen.\n",
+      "Lets create our first Figure object using pyplot, and then show it:"
```

Comment:
Does it matter that Axes is technically a plural and I will be referring to it in the singular?  This is the main reason I have used the word object so much.